test/components/Meter.test.tsx

```diff
@@ -31,10 +31,9 @@ const exampleProps = {
   style,
   value: 25,
   min: 0,
-  max: 50,
+  max: 100,
```

Comment:
this is the fix that required the least effort. Tweaking the ranges would've been more invasive.